firmware/console/binary/tunerstudio_outputs.h

```diff
@@ -181,17 +181,22 @@ typedef struct {
 	TsCalMode calibrationMode; // 132
 	uint8_t padding[1]; // 133
 
+	// Values used for load axes for fuel/ign tables
+	// These may or may not be the same value, depending on mode
```

Comment:
i see no change to rusEFI console which at the moment has all these offsets hard-coded

would it be better to add new fields into the end of the struct?

Reply:
it's a draft pr 🙂 

These can't be too far from the beginning because TS will read them every time, and we want them in the big contiguous block at the beginning